### ISSUE TITLE
Nodepower reports suspicious values on the aggregator node

### ISSUE THREAD

**Issue by noahv** [open-mpi/orcm#75]
For some reason, the headnode is reporting a very low 3.9 for nodepower:

<pre>
2014-09-29 16:37:28-0700,node05,162.092102
2014-09-29 16:37:28-0700,node22,156.293335
2014-09-29 16:37:28-0700,node20,158.253326
2014-09-29 16:37:28-0700,node28,162.186661
2014-09-29 16:37:32-0700,headnode,3.906667
2014-09-29 16:37:33-0700,node17,162.000000
2014-09-29 16:37:33-0700,node04,162.266663
2014-09-29 16:37:33-0700,node23,162.493332
2014-09-29 16:37:33-0700,node03,177.093338
2014-09-29 16:37:33-0700,node08,160.466660
2014-09-29 16:37:33-0700,node01,195.093338
2014-09-29 16:37:33-0700,node21,159.693329
</pre>

**Comment by noahv** [comment]
Issue is still present:

<pre>
/opt/open-rcm/bin/orcmd --omca sensor_base_sample_rate 5 --omca sensor_heartbeat_rate 10 --omca sensor_base_log_samples 1 --omca db_p
rint_file - --omca sensor heartbeat,nodepower
[headnode:118282] 
******************************
Thu Jan 29 07:52:09 2015: ORCM version: 0.7.0rc2 AGGREGATOR: [[0,0],1] started and connected to AGGREGATOR: [[0,0],0]
******************************
 
DB request: store; data:
primary_key=nodepower,ctime=2015-01-29 07:52:14-0800,hostname=headnode,nodepower=2.640000:W
DB request: store; data:
primary_key=nodepower,ctime=2015-01-29 07:52:19-0800,hostname=headnode,nodepower=95.306664:W
</pre>

**Comment by vpedabal** [comment]
Removing the release blocker label, and record in the release notes.

**Comment by forzaclaudio** [comment]
After some longevity testing we were able to spot a behaviour that might be and indicationg of a bug related with the way nodepower component obtains its values as shown in the following plots. 
![headnodeorcmdb-nodepower](https://cloud.githubusercontent.com/assets/8369713/6755601/f5288928-cee9-11e4-8b7b-da1430e605d5.png)
**Figure 1.** Data written into the DB during ORCM execution (aggregator daemon) for more that 60 hours. Time period of the sampling rate was (120 seconds) and we can observe that the drop in readings of nodepower values to approximately 2.9 Watts appears with a frequency of 5,520 seconds (~ 92 mins, 46 sampling events) when the node usage corresponds to that an **IDLE** state

We have also observed a similar behaviour in other nodes but the reading drops ocurred to what appear to be three different and clearly distinguishable levels.

![node01orcmdb-nodepower](https://cloud.githubusercontent.com/assets/8369713/6755943/33ab0bd8-ceec-11e4-8d04-12ae8086d0fc.png)
**Figure 2.** Data written into the DB during ORCM execution (aggregator daemon) for more that 60 hours. Time period of the sampling rate was (120 seconds) and we can observe that the drop in readings of nodepower to values of -2 Watts appears with a frequency of 135,840 (~ 37.73 hours, 1132 sampling events). Another drop of readings to values of approximately 171 Watts appears with a frequency of 2,880 seconds (48 mins, 24 sampling events). A third drop of readings to values of approximately 192 Watts appears with a frequency of 3,240 seconds (54 mins, 27 sampling events). All of the above drops occur when node usage correspond to that of an **IDLE** state

There is also a possibility that ORCM has no bug at all and what we are observing has an origin in the hardware e.g. PSU, chassis, etc where the tests were performed. Because there is one node on which this behaviour is not present, as shown in the following figure.

![node07orcmdb-nodepower](https://cloud.githubusercontent.com/assets/8369713/6756909/ae69edfc-cef2-11e4-82d3-30ca09941427.png)
**Figure 3.** Data written into the DB during ORCM execution (aggregator daemon) for more that 60 hours. Time period of the sampling rate was (120 seconds) and we can see that there were no drops in power readings while the node state was **IDLE**. However power drops were observed during the time where the node was executing a HPC workload during the first 11280 seconds (188 mis).

These tests were performed in a system with more than 30 nodes. A similar behaviour to the ones depicted in Figures 1 and 2 were observed in all but one of the nodes. The exceptional case is shown in Figure 3.
